src/Components/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const button = (props) => <div className="button"> { props.name } </div>;
+
+button.propTypes = { name: PropTypes.string };
+
```

Comment:
- [ ] Since it has no default value, prop name should be required (Eslint will complain about it)
button.propTypes = { name: PropTypes.string.isRequired };